cyral/internal/sidecar/listener/schema_loader.go

```diff
@@ -0,0 +1,31 @@
+package listener
+
+import (
+	"github.com/cyralinc/terraform-provider-cyral/cyral/core"
+)
+
+type packageSchema struct {
+}
+
+func (p *packageSchema) Name() string {
+	return "sidecar"
```

Comment:
```suggestion
	return "listener"
```

Reply:
Thanks! Addressed in 1d4545f311c3daf857fc7a7ab532fac26aa70c62.